### ISSUE TITLE
Web page for viewing contact request

### ISSUE THREAD

**Issue by grundic** [opensourcehacker/sevabot#18]
Add web page for administrator for viewing and accepting/rejecting incoming contact requests.
Depends on https://github.com/opensourcehacker/sevabot/issues/17

**Comment by miohtama** [comment]
Yes. This would enable to make bot to join group chats without going to the headless server through VNC.

You still cannot automatize Skype install and first run, though :(

**Comment by grundic** [comment]
> > You still cannot automatize Skype install and first run, though :(
> > Well, i've wrote simple init.d script that just runs scripts/{start-server.sh start-vnc.sh}.
> > So, in this case server is configured to automatic service startup.

But first installation, script connection with Skype have to be done manually. But just once! :)
